scripts/xe-toolstack-restart

```diff
@@ -32,6 +32,13 @@ SERVICES="perfmon xapi v6d xenopsd xenopsd-xc xenopsd-xenlight
   xcp-rrdd-xenpm xcp-rrdd-gpumon xcp-rrdd xcp-networkd squeezed forkexecd
   $MPATHALERT xapi-storage-script xapi-clusterd varstored-guard message-switch"
 
+# the following command gives ugly output
+/opt/xensource/libexec/xapissl stop > /dev/null 2>&1
+kill_stunnel_exit_code=$?
+if [[ $kill_stunnel_exit_code != 0 ]]; then
+        echo "Failed to kill stunnel processes!";
+fi
```

Comment:
If the command failed, we have no idea why. Maybe:
```
ERROR=$(/opt/xensource/libexec/xapissl stop > /dev/null 2>&1)
if [[ $? != 0 ]]; then
   echo "Failed to kill stunnel processes!"
   echo "$ERROR"
fi
```

I don't have enough experience with shell scripts to know whether that is a viable idea.

Reply:
i'll test this

Reply:
I think the redirections need to be in the opposite order, otherwise it all goes to `/dev/null`: https://stackoverflow.com/a/549776

Reply:
that actually was my intention because the output is messy

instead it would be better to store stdout and stderr somewhere, and display them if there was a problem